### DISCUSSION TITLE
AMD - Errors   1.1.431 +

### DISCUSSION THREAD

**Discussion by SunGreen777** [Mikubill/sd-webui-controlnet#2545] [General]
Hello, starting from version 1.1.431 the IP adapter on Direct ML stopped working, the problem is multiple.
Please check it.
A clean install doesn't help either.

**Comment by huchenlei** [comment]
Please file a bug under /issues with console output. Thanks!

**Reply by huchenlei** [reply]
We recently make CLIP run on your GPU instead of CPU by default. You can switch this setting back according to steps in https://github.com/Mikubill/sd-webui-controlnet/pull/2495.
